data/json/recipes/recipe_food.json

```diff
@@ -4228,17 +4229,24 @@
 },{
   "type" : "recipe",
   "result": "syrup",
+  "id_suffix": "with_instrument",
```

Comment:
Adding a separate recipe just for a tiny bit lower cost is bloat.

Wait with it until we get optional tools.